### ISSUE TITLE
Incorrectly generated flows

### ISSUE THREAD

**Issue by sobek1886** [MouseLand/cellpose#799]
Hello.
I'm trying to fine-tune the cyto model.
Before encountering Cellpose, I had labeled my data using Label Studio and created binary PNG masks. I only have 1 binary mask for each image (so no separate masks for each of the 'cells' inside an image).

This is how my example mask and image look like.

![0155_coax_D__9_patch_33_mask](https://github.com/MouseLand/cellpose/assets/75131061/b8a42ea5-a8ee-488f-bff9-027734f1c0f4)
![0155_coax_D__9_patch_33](https://github.com/MouseLand/cellpose/assets/75131061/d0dd06ee-6755-454c-9b25-4ff3bfc8c96d)

Note I'm using absolute paths, didn't want to include them here
When trying to train the model using this code in my terminal:

'''
python -m cellpose --verbose --train --dir (...)/train_separate_labels --pretrained_model cyto --chan 0 --chan2 0 --n_epochs 100 --min_train_masks 1 --learning_rate 0.1 --weight_decay 0.0001 --test_dir (...)/test_separate_labels --mask_filter _mask
'''

Everything seems to go well:
'''
2023-10-05 11:12:56,849 [INFO] WRITING LOG OUTPUT TO (...)/run.log
2023-10-05 11:12:56,849 [INFO] 
cellpose version:       2.2.3 
platform:               darwin 
python version:         3.9.18 
torch version:          2.0.1
2023-10-05 11:12:56,849 [INFO] >>>> using CPU
2023-10-05 11:12:56,877 [INFO] not all flows are present, running flow generation for all images
2023-10-05 11:12:57,331 [INFO] 490 / 490 images in (...)/train_patches folder have labels
2023-10-05 11:12:57,357 [INFO] not all flows are present, running flow generation for all images
2023-10-05 11:13:01,394 [INFO] 470 / 470 images in (...)/test_patches folder have labels
2023-10-05 11:13:01,395 [INFO] >>>> during training rescaling images to fixed diameter of 30.0 pixels
2023-10-05 11:13:01,395 [INFO] >> cyto << model set to be used
2023-10-05 11:13:01,395 [INFO] WARNING: MKL version on torch not working/installed - CPU version will be slightly slower.
'''

**But all the generated flows are completely black.**

I think this is why the model is not able to detect any ROIs when I try to run it in the GUI after training.
Not sure why is that happening and how to fix it.

I saw that someone had a similar problem in issue #446 but it is unclear to me how they solved it.

**Comment by mrariden** [comment]
Cellpose expects masks that are:

- uint8 or uint16
- 0 background
- instance labeled (cell 1 has all 1's mask, cell 2 has all 2's mask)
- no gaps in instance labels

If you data doesn't fit the requirements it might not train correctly.

**Comment by mrariden** [comment]
closing due to inactivity
